css-view-transitions-1/Overview.bs

```diff
@@ -1565,6 +1572,8 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		1. [=Assert=]: |transition|'s [=ViewTransition/phase=] is not "`done`".
 
+		1. If |transition|'s [=ViewTransition/phase=] is "`pending-capture`" and |transition|'s [=ViewTransition/process old state captured=] is not null, then call [=ViewTransition/process old state captured=] and return.
```

Comment:
Shouldn't the phase value be updated before calling "process old state captured"? Otherwise that algorithm will incorrectly assume that the capture was successful.

Doing this as the last step is a good idea.

Reply:
Fixed